etcdctlv3/command/util.go

```diff
@@ -47,3 +50,9 @@ func argify(s string) []string {
 	r := regexp.MustCompile("'.+'|\".+\"|\\S+")
 	return r.FindAllString(s, -1)
 }
+
+func contextWithTotalTimeout(cmd *cobra.Command) (context.Context, context.CancelFunc) {
+	var timeOut time.Duration
+	cmd.Flags().DurationVar(&timeOut, "total-timeout", 0, "")
```

Comment:
"snaphot" and "txn" command seems time-consuming, is 5 seconds safe enough? Not familiar with "snaphot" and "txn" command though, just my intuition.